_data/simple-icons.json

```
@@ -7708,6 +7708,12 @@
             "hex": "8ED500",
             "source": "https://about.nextdoor.com/us-media/"
         },
+        {
+            "title": "Nextra",
+            "hex": "000000",
+            "source": "https://nextra.site/",
```

```suggestion
            "source": "https://nextra.site",
```

Remove redundant trailing slash